driver/flambda_backend_flags.ml

```diff
@@ -58,6 +58,16 @@ let flags_by_opt_level ~opt_level ~default ~oclassic ~o2 ~o3 =
   | Set O3 -> o3
 
 module Flambda2 = struct
+  let debug = (* -flambda2-debug *)
+    let default =
+      match Sys.getenv "FLAMBDA2_DEBUG" with
```

Comment:
Hmm, I'd rather not have the compiler read more environment variables.  You should be able to arrange for this to work via `OCAMLPARAM` I think?